src/arrayMethodSplice.js

```diff
@@ -4,8 +4,29 @@
  * Implement method Splice
  */
 function applyCustomSplice() {
-  [].__proto__.splice2 = function(start, deleteCount, ...items) {
-    // write code here
+  [].__proto__.splice2 = function(
+    start = 0, deleteCount = this.length, ...items) {
+    const resultSplice = [];
+    const partAfterDelete = [];
+    let deleteFrom = start;
+    let deleteNumber = deleteCount;
+    if (start < 0) { deleteFrom = start + this.length; }
```

Comment:
write body of `if` blocks on the separate line + don't forget to set line breaks between logical blocks

Reply:
> write body of `if` blocks on the separate line + don't forget to set line breaks between logical blocks

fixed

Reply:
this comments isn't fully followed

Reply:
> this comments isn't fully followed

fixed